src/WebJobs.Script/Binding/ManualBinding.cs

```diff
@@ -1,29 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.ObjectModel;
-using System.IO;
-using System.Reflection.Emit;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Script.Description;
-
-namespace Microsoft.Azure.WebJobs.Script.Binding
-{
-    public class ManualBinding : FunctionBinding
```

Comment:
This was a stub I had put in recently not realizing that a more comprehensive fix was needed.